src/createDir.ts

```diff
@@ -0,0 +1,13 @@
+import { existsSync, mkdirsSync } from 'fs-extra';
+import * as chalk from 'chalk';
+
+export default function (...dirPath: string[]) {
+	dirPath.forEach((path) => {
```

Comment:
do you have to check if the directory exists for each one? couldn't you do this once for the parent directory?